content/master/cli/command-reference.md

```diff
@@ -856,135 +851,182 @@ A Kubernetes cluster running Crossplane isn't required.
+To clear the cache and download the CRD files again use the `--clean-cache` flag.
+
+For example, to validate a manage resource against a provider,
+first, create a provider manifest file. For example, to validate an IAM role
+from Provider AWS, use the 
+[Provider AWS IAM](https://marketplace.upbound.io/providers/upbound/provider-aws-iam/v1.0.0) 
+manifest.
+
+{{<hint "tip" >}}
+To validate a 
+"[family provider](https://blog.upbound.io/new-provider-families)" use the
+provider manifests of the resources to validate.
+{{< /hint >}}
+
+```yaml
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-aws-iam
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-iam:v1.0.0
 ```
-{{< /expand >}}
 
-#### Validate render command output
+Now include the XR or managed resource to validate.
 
-Passing the output of the _render_ command to the _validate_ command. 
+For example, to validate an 
+{{<hover label="iamAK" line="2">}}AccessKey{{</hover>}} managed resource,
+provide a managed resource YAML file. 
 
-`crossplane beta render xr.yaml composition.yaml func.yaml | crossplane beta validate schemas.yaml -`
+```yaml {label="iamAK"}
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: AccessKey
+metadata:
+  name: sample-access-key-0
+spec:
+  forProvider:
+    userSelector:
+      matchLabels:
+        example-name: test-user-0
+```
 
-{{< expand "Pipe render result to validate command" >}}
-```console
-[x] schema validation error example.crossplane.io/v1beta1, Kind=XR, example : status.conditions[0].lastTransitionTime: Invalid value: "null": status.conditions[0].lastTransitionTime in body must be of type string: "null"
-[x] schema validation error example.crossplane.io/v1beta1, Kind=XR, example : spec: Required value
+Run the `crossplane beta validate` command providing the provider and managed
+resource YAML files as input. 
+
+```shell
+crossplane beta validate provider.yaml managedResource.yaml
 [✓] iam.aws.upbound.io/v1beta1, Kind=AccessKey, sample-access-key-0 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=AccessKey, sample-access-key-1 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=User, test-user-0 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=User, test-user-1 validated successfully
-Total 5 resources: 0 missing schemas, 4 success cases, 1 failure cases
+Total 1 resources: 0 missing schemas, 1 success case, 0 failure cases
 ```
-{{< /expand >}}
 
-Make sure to include full XR in the render output to avoid the _missing resource_ warning.
 
-`crossplane beta render xr.yaml composition.yaml func.yaml --include-full-xr | crossplane beta validate schemas.yaml -`
+#### Validate render command output
+
+You can pipe the output of `crossplane beta render` into 
+`crossplane beta validate` to validate complete Crossplane resource pipelines,
+including XRs, compositions and composition functions. 
+
+Use the `--include-full-xr` command with `crossplane beta render` and the `-` 
```

Comment:
I'm not sure how to reword it, but the `-` operator is receiving the output from `stdin`. Technically, the output of `crossplane beta render` is passed as input to stdin to the `crossplane beta validate` command. I'm also ok to keep it as it, because maybe it's too much detail.

Maybe something like this:
```text
Use the --include-full-xr flag of the crossplane beta render. command to pass the output directly to crossplane beta validate, use the - option, which reads from stdin. This effectively pipes the output of crossplane beta render as input to crossplane beta validate.
```